DESCRIPTION

```diff
@@ -23,7 +28,12 @@ Encoding: UTF-8
 LazyData: true
 biocViews: Software, Infrastructure
 Imports:
+    BiocManager,
     Biostrings,
+    GOSim,
+    org.Hs.eg.db,
```

Comment:
Cathy - I won't merge that in. Adding `org.Hs.eg.db` to `Imports:` is a **huge** burden on the package: this is a 310 MB sqlite database! It would need to be freshly imported and downloaded for every check, and every fresh installation! Such packages **must** be installed explicitly on a per-need basis.

You should write an exported helper function that could look like this and that would only be executed when someone **actually needs** the database. (This code is however untested):
```R
installHumanGenomeAnnotation <- function() {
  if (! requireNamespace("org.Hs.eg.db")) {
    BiocManager::install("org.Hs.eg.db")
  }
  return(invisible(NULL))
}
```
Also make sure that all of your examples are protected in `dontrun{ ... }` blocks.

After you have made the required changes, I will re-review.

Reply:
Created helper function HuGenAnnotImport.R import the annotation database only when the semanticSimilarity function is run 

- Thank you